WNPRC_Purchasing/test/src/org/labkey/test/tests/wnprc_purchasing/WNPRC_PurchasingTest.java

```diff
@@ -0,0 +1,188 @@
+ * limitations under the License.
+ */
+
+package org.labkey.test.tests.wnprc_purchasing;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.remoteapi.Connection;
+import org.labkey.remoteapi.query.InsertRowsCommand;
+import org.labkey.remoteapi.query.SaveRowsResponse;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.CustomModules;
+import org.labkey.test.categories.EHR;
+import org.labkey.test.util.APIUserHelper;
+import org.labkey.test.util.AbstractUserHelper;
+import org.labkey.test.util.PostgresOnlyTest;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.labkey.test.WebTestHelper.getRemoteApiConnection;
+
+@Category({CustomModules.class, EHR.class})
+public class WNPRC_PurchasingTest extends BaseWebDriverTest implements PostgresOnlyTest
+{
+    private static final String FOLDER_TYPE = "WNPRC Purchasing";
+
+    private static final String REQUESTER_USER = "purchaserequester@test.com";
+    private String _adminUserId;
+
+    private static final String ADMIN_USER = "purchaseadmin@test.com";
+    private String _requesterUserId;
+
+    private final File ALIASES_TSV = TestFileUtils.getSampleData("wnprc_purchasing/aliases.tsv");
+    private final File ITEM_UNITS_TSV = TestFileUtils.getSampleData("wnprc_purchasing/itemUnits.tsv");
+    private final File LINE_ITEM_STATUS_TSV = TestFileUtils.getSampleData("wnprc_purchasing/lineItemStatus.tsv");
+    private final File SHIPPING_INFO_TSV = TestFileUtils.getSampleData("wnprc_purchasing/shippingInfo.tsv");
+    private final File VENDOR_TSV = TestFileUtils.getSampleData("wnprc_purchasing/vendor.tsv");
+
+    private final String ACCT_100 = "acct100";
+    private final String ACCT_101 = "acct101";
+
+    public AbstractUserHelper _userHelper = new APIUserHelper(this);
+
+    @Override
+    protected void doCleanup(boolean afterTest) throws TestTimeoutException
+    {
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    @BeforeClass
+    public static void setupProject() throws IOException, CommandException
+    {
+        WNPRC_PurchasingTest init = (WNPRC_PurchasingTest) getCurrentTest();
+
+        init.doSetup();
+    }
+
+    private void doSetup() throws IOException, CommandException
+    {
+        log("Creating a 'WNPRC Purchasing' folder");
+        _containerHelper.createProject(getProjectName(), FOLDER_TYPE);
+
+        log("Creating a purchasing admin user");
+        _adminUserId = _userHelper.createUser(ADMIN_USER).getUserId().toString();
```

Comment:
Make the _adminUserId and _requesterUserId fields int instead of String, and use getUserId().intValue().

Reply:
good point, done!